packages/components/form-field/_form-field-theme.scss

```
@@ -1,5 +1,17 @@
 @use '../core/styles/common/tokens';
 
+@mixin kbq-form-field-hint-theme {
```

если это приватная миксина
```suggestion
@mixin _kbq-form-field-hint-theme {
```

---

не приватная